torchci/pages/api/drci/drci.ts

```diff
@@ -85,18 +85,26 @@ export default async function handler(
 export async function updateDrciComments(
   octokit: Octokit,
   repo: string = "pytorch",
-  prNumber?: string
+  prNumbers: number[]
 ): Promise<{ [pr: number]: { [cat: string]: RecentWorkflowsData[] } }> {
-  const recentWorkflows: RecentWorkflowsData[] = await fetchRecentWorkflows(
-    `${OWNER}/${repo}`,
-    prNumber,
-    NUM_MINUTES + ""
-  );
+  const [recentWorkflows, workflowsFromPendingComments] = await Promise.all([
+    // Fetch in two separate queries because combinidng into one query took much
+    // longer to run on CH
+    fetchRecentWorkflows(`${OWNER}/${repo}`, prNumbers, NUM_MINUTES + ""),
+    fetchRecentWorkflows(
+      `${OWNER}/${repo}`,
+      // Only fetch if we are not updating a specific PR
+      prNumbers.length == 0
```

Comment:
When someone explicitly calls `pytorbot drci` that implies there's some out of sync jobs.  That would be an important time to go further back in history to get the correct job status.

Reply:
I'm a little confused as to what the problem is here?  If someone calls pytorchbot drci, then it would be handled by the other call to fetchRecentWorkflows and the num minutes doesn't get used there are prnumbers in the list